Router.svelte

```diff
@@ -76,7 +76,7 @@ export function push(location) {
 
     // Execute this code when the current call stack is complete
     setTimeout(() => {
-        window.location.hash = '#' + location
+        window.location.hash = location
```

Comment:
This might be breaking. Should add an “if” statement to add a # if not present already.

Reply:
Ah yeah, if function is used in code it can break. So i just concat a # if not present.